### ISSUE TITLE
Headswaps

### ISSUE THREAD

**Issue by keyten** [Razish/japp#279]
Like in the redsaurus' sp customization mod :)
http://jkhub.org/topic/2480-sp-customisation-stuff/

**Comment by Razish** [comment]
Only you will be able to see it, AFAIK, and it may not work on all servers.
Low priority, @redsaurus may want to submit a patch for it

**Comment by keyten** [comment]
[2:21:48] EpicLoyd: https://github.com/Razish/japp/issues/279
[2:21:57] EpicLoyd: say him that we can send data to server
[2:22:57] EpicLoyd: ./lua SendReliableCommand(-1 '0 head_mando 1 head_lando 2 head_kyle')
[2:23:10] EpicLoyd: id (head) id (head)

**Comment by Razish** [comment]
Yes,  but the most ideal solution would be one that works without modifying the server, so you'd be forced to pack it in unrelated fields and hope they're still parsed correctly, and it all becomes a mess.
I don't like toying with things that break compatibility and/or require a client/server mod to work nicely

**Comment by EpicLoyd** [comment]
we can assign id to heads, and transmit them, using:
	int			userInt1;
	int			userInt2;
	int			userInt3;

**Comment by Razish** [comment]
Nope, those fields are networked as 1-bit. Changing that _forces_ a client mod

**Comment by EpicLoyd** [comment]
the only solution is to send through sendreliablecommand?

**Comment by Razish** [comment]
Yes, or pack it into userinfo fields and hope everything is parsed correctly on different mods.
Having a CSF (client support flag) to indicate they support the reliable command is probably the best approach, but as I said it's low on my priority list. It would be good if @redsaurus could submit a patch to actually support the headswapping and I could add the support flag side of things.
